src/styles/layout/index.md.njk

```diff
@@ -1,12 +1,14 @@
 ---
 title: Layout
-description:
+description: Organise the layout of the page into blocks
 section: Styles
 aliases:
 backlog_issue_id:
 layout: layout-pane.njk
 ---
 
+Organise the layout of the page into blocks
```

Comment:
Could this come out for now? Amy said that she'd like to be consulted over any changes to page intros